engine/installation/linux/docker-ce/debian.md

```diff
@@ -208,16 +208,15 @@ from the repository.
     ```bash
     $ apt-cache madison docker-ce
 
-    docker-ce | {{ minor-version }}.0~ce-0~debian-jessie | {{ download-url-base}} jessie/stable amd64 Packages
+    docker-ce | {{ minor-version }}.0~ce-0~debian | {{ download-url-base}} jessie/stable amd64 Packages
```

Comment:
Yeah that would probably be the most transparent way to do it. But will it break people who have already pinned? Can we keep the packages there with the old names but have them with the new names too?